CRITICAL - 2024-03-11 09:52:56 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
in APPPATH\Helpers\jwt_helper.php on line 23.
 1 APPPATH\Filters\JWTAuthenticationFilter.php(34): validateJWTFromRequest('eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InJhdWxAZXhhbXBsZS5jb20iLCJpYXQiOjE3MDk5MzIzMzIsImV4cCI6MTcxMTczMjMzMn0.k0PQQ0NTBpK9Ux68aoz3cfTDikR_iKKipuyTiOEMP1U')
 2 SYSTEMPATH\Filters\Filters.php(184): App\Filters\JWTAuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\CodeIgniter.php(474): CodeIgniter\Filters\Filters->run('client', 'before')
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 09:57:37 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
in APPPATH\Helpers\jwt_helper.php on line 23.
 1 APPPATH\Filters\JWTAuthenticationFilter.php(34): validateJWTFromRequest('eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InJhdWxAZXhhbXBsZS5jb20iLCJpYXQiOjE3MDk5MzIzMzIsImV4cCI6MTcxMTczMjMzMn0.k0PQQ0NTBpK9Ux68aoz3cfTDikR_iKKipuyTiOEMP1U')
 2 SYSTEMPATH\Filters\Filters.php(184): App\Filters\JWTAuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\CodeIgniter.php(474): CodeIgniter\Filters\Filters->run('client', 'before')
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 09:58:37 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
in APPPATH\Helpers\jwt_helper.php on line 23.
 1 APPPATH\Filters\JWTAuthenticationFilter.php(34): validateJWTFromRequest('eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InJhdWxAZXhhbXBsZS5jb20iLCJpYXQiOjE3MDk5MzIzMzIsImV4cCI6MTcxMTczMjMzMn0.k0PQQ0NTBpK9Ux68aoz3cfTDikR_iKKipuyTiOEMP1U')
 2 SYSTEMPATH\Filters\Filters.php(184): App\Filters\JWTAuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\CodeIgniter.php(474): CodeIgniter\Filters\Filters->run('client', 'before')
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 09:59:51 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
in APPPATH\Helpers\jwt_helper.php on line 23.
 1 APPPATH\Filters\JWTAuthenticationFilter.php(34): validateJWTFromRequest('eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InJhdWxAZXhhbXBsZS5jb20iLCJpYXQiOjE3MDk5MzIzMzIsImV4cCI6MTcxMTczMjMzMn0.k0PQQ0NTBpK9Ux68aoz3cfTDikR_iKKipuyTiOEMP1U')
 2 SYSTEMPATH\Filters\Filters.php(184): App\Filters\JWTAuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\CodeIgniter.php(474): CodeIgniter\Filters\Filters->run('client', 'before')
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 10:09:01 --> Error: Class "App\Controllers\ClientModel" not found
in APPPATH\Controllers\Client.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Client->index()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Client))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 10:48:11 --> CodeIgniter\Database\Exceptions\DatabaseException: You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 884.
 1 SYSTEMPATH\Model.php(662): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(864): CodeIgniter\Model->builder()
 3 APPPATH\Models\UserModel.php(16): CodeIgniter\Model->__call('where', [...])
 4 APPPATH\Validation\UserRules.php(11): App\Models\UserModel->findUserByEmailAddress('raul@example.com')
 5 SYSTEMPATH\Validation\Validation.php(328): App\Validation\UserRules->validateUser('12345678', 'email, password', [...], null, 'contraseña')
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('contraseña', 'contraseña', '12345678', [...], [...])
 7 APPPATH\Controllers\BaseController.php(66): CodeIgniter\Validation\Validation->run([...])
 8 APPPATH\Controllers\Auth.php(21): App\Controllers\BaseController->validateRequest([...], [...], [...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 10:54:54 --> CodeIgniter\Database\Exceptions\DatabaseException: You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 884.
 1 SYSTEMPATH\Model.php(662): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(864): CodeIgniter\Model->builder()
 3 APPPATH\Models\UserModel.php(16): CodeIgniter\Model->__call('where', [...])
 4 APPPATH\Validation\UserRules.php(11): App\Models\UserModel->findUserByEmailAddress('raul@example.com')
 5 SYSTEMPATH\Validation\Validation.php(328): App\Validation\UserRules->validateUser('12345678', 'email, password', [...], null, 'contraseña')
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('contraseña', 'contraseña', '12345678', [...], [...])
 7 APPPATH\Controllers\BaseController.php(66): CodeIgniter\Validation\Validation->run([...])
 8 APPPATH\Controllers\Auth.php(21): App\Controllers\BaseController->validateRequest([...], [...], [...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 14:00:46 --> TypeError: App\Controllers\Users::auth(): Return value must be of type CodeIgniter\HTTP\ResponseInterface, none returned
in APPPATH\Controllers\Users.php on line 21.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Users->auth()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 15:11:49 --> ParseError: syntax error, unexpected token ";", expecting ")"
in APPPATH\Controllers\BaseController.php on line 57.
 1 SYSTEMPATH\Autoloader\Autoloader.php(290): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\apisolve\\app\\Controllers\\BaseController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\BaseController')
 3 APPPATH\Controllers\Users.php(9): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\BaseController')
 4 SYSTEMPATH\Autoloader\Autoloader.php(311): include_once('C:\\xampp\\htdocs\\apisolve\\app\\Controllers\\Users.php')
 5 SYSTEMPATH\Autoloader\Autoloader.php(290): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\apisolve\\app\\Controllers\\Users.php')
 6 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Users')
 7 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Users')
 8 SYSTEMPATH\CodeIgniter.php(899): class_exists('\\App\\Controllers\\Users', true)
 9 SYSTEMPATH\CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
ERROR - 2024-03-11 16:09:29 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@example.com and password = 12345678 and token = 1 and active = 1' at line 1 in C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT user, pa...', 0)
#1 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT user, pa...')
#2 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT user, pa...')
#3 C:\xampp\htdocs\apisolve\app\Models\UserModel.php(34): CodeIgniter\Database\BaseConnection->query('SELECT user, pa...')
#4 C:\xampp\htdocs\apisolve\app\Validation\UserRules.php(12): App\Models\UserModel->authenticateToken('raul@example.co...', '12345678')
#5 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Validation\Validation.php(328): App\Validation\UserRules->validateUser('12345678', 'email, password', Array, NULL, 'contrase\xC3\xB1a')
#6 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('contrase\xC3\xB1a', 'contrase\xC3\xB1a', '12345678', Array, Array)
#7 C:\xampp\htdocs\apisolve\app\Controllers\BaseController.php(66): CodeIgniter\Validation\Validation->run(Array)
#8 C:\xampp\htdocs\apisolve\app\Controllers\Users.php(21): App\Controllers\BaseController->validateRequest(Array, Array, Array)
#9 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\CodeIgniter.php(941): App\Controllers\Users->auth()
#10 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\apisolve\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-03-11 16:09:29 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@example.com and password = 12345678 and token = 1 and active = 1' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 APPPATH\Models\UserModel.php(34): CodeIgniter\Database\BaseConnection->query('SELECT user, password FROM apisolve_sandbox.users WHERE user = raul@example.com and password = 12345678 and token = 1 and active = 1')
 2 APPPATH\Validation\UserRules.php(12): App\Models\UserModel->authenticateToken('raul@example.com', '12345678')
 3 SYSTEMPATH\Validation\Validation.php(328): App\Validation\UserRules->validateUser('12345678', 'email, password', [...], null, 'contraseña')
 4 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('contraseña', 'contraseña', '12345678', [...], [...])
 5 APPPATH\Controllers\BaseController.php(66): CodeIgniter\Validation\Validation->run([...])
 6 APPPATH\Controllers\Users.php(21): App\Controllers\BaseController->validateRequest([...], [...], [...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Users->auth()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 16:09:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@example.com and password = 12345678 and token = 1 and active = 1' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT user, password FROM apisolve_sandbox.users WHERE user = raul@example.com and password = 12345678 and token = 1 and active = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT user, password FROM apisolve_sandbox.users WHERE user = raul@example.com and password = 12345678 and token = 1 and active = 1')
 3 APPPATH\Models\UserModel.php(34): CodeIgniter\Database\BaseConnection->query('SELECT user, password FROM apisolve_sandbox.users WHERE user = raul@example.com and password = 12345678 and token = 1 and active = 1')
 4 APPPATH\Validation\UserRules.php(12): App\Models\UserModel->authenticateToken('raul@example.com', '12345678')
 5 SYSTEMPATH\Validation\Validation.php(328): App\Validation\UserRules->validateUser('12345678', 'email, password', [...], null, 'contraseña')
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('contraseña', 'contraseña', '12345678', [...], [...])
 7 APPPATH\Controllers\BaseController.php(66): CodeIgniter\Validation\Validation->run([...])
 8 APPPATH\Controllers\Users.php(21): App\Controllers\BaseController->validateRequest([...], [...], [...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Users->auth()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 16:09:29 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@example.com and password = 12345678 and token = 1 and active = 1' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT user, password FROM apisolve_sandbox.users WHERE user = raul@example.com and password = 12345678 and token = 1 and active = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT user, password FROM apisolve_sandbox.users WHERE user = raul@example.com and password = 12345678 and token = 1 and active = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT user, password FROM apisolve_sandbox.users WHERE user = raul@example.com and password = 12345678 and token = 1 and active = 1')
 4 APPPATH\Models\UserModel.php(34): CodeIgniter\Database\BaseConnection->query('SELECT user, password FROM apisolve_sandbox.users WHERE user = raul@example.com and password = 12345678 and token = 1 and active = 1')
 5 APPPATH\Validation\UserRules.php(12): App\Models\UserModel->authenticateToken('raul@example.com', '12345678')
 6 SYSTEMPATH\Validation\Validation.php(328): App\Validation\UserRules->validateUser('12345678', 'email, password', [...], null, 'contraseña')
 7 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('contraseña', 'contraseña', '12345678', [...], [...])
 8 APPPATH\Controllers\BaseController.php(66): CodeIgniter\Validation\Validation->run([...])
 9 APPPATH\Controllers\Users.php(21): App\Controllers\BaseController->validateRequest([...], [...], [...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Users->auth()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 16:42:02 --> ErrorException: Undefined array key "user"
in APPPATH\Validation\UserRules.php on line 16.
 1 APPPATH\Validation\UserRules.php(16): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "user"', 'C:\\xampp\\htdocs\\apisolve\\app\\Validation\\UserRules.php', 16)
 2 SYSTEMPATH\Validation\Validation.php(328): App\Validation\UserRules->validateUser('12345678', 'usuario, contraseña', [...], null, 'contraseña')
 3 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('contraseña', 'contraseña', '12345678', [...], [...])
 4 APPPATH\Controllers\BaseController.php(66): CodeIgniter\Validation\Validation->run([...])
 5 APPPATH\Controllers\Users.php(22): App\Controllers\BaseController->validateRequest([...], [...], [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Users->auth()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 17:04:54 --> Error: Call to undefined method App\Controllers\Users::getJWTForUser()
in APPPATH\Controllers\Users.php on line 25.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Users->auth()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 17:06:24 --> ErrorException: Undefined array key "email"
in APPPATH\Controllers\Auth.php on line 24.
 1 APPPATH\Controllers\Auth.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "email"', 'C:\\xampp\\htdocs\\apisolve\\app\\Controllers\\Auth.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
ERROR - 2024-03-11 17:09:40 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@example.com and active = 1' at line 1 in C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT * FROM a...', 0)
#1 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM a...')
#2 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM a...')
#3 C:\xampp\htdocs\apisolve\app\Models\UserModel.php(41): CodeIgniter\Database\BaseConnection->query('SELECT * FROM a...')
#4 C:\xampp\htdocs\apisolve\app\Helpers\jwt_helper.php(25): App\Models\UserModel->findUserByEmailAddress('raul@example.co...')
#5 C:\xampp\htdocs\apisolve\app\Filters\JWTAuthenticationFilter.php(34): validateJWTFromRequest('eyJ0eXAiOiJKV1Q...')
#6 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Filters\Filters.php(184): App\Filters\JWTAuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#7 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\CodeIgniter.php(474): CodeIgniter\Filters\Filters->run('client/', 'before')
#8 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\apisolve\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\apisolve\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-03-11 17:09:40 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@example.com and active = 1' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 APPPATH\Models\UserModel.php(41): CodeIgniter\Database\BaseConnection->query('SELECT * FROM apisolve_sandbox.users WHERE email = raul@example.com and active = 1')
 2 APPPATH\Helpers\jwt_helper.php(25): App\Models\UserModel->findUserByEmailAddress('raul@example.com')
 3 APPPATH\Filters\JWTAuthenticationFilter.php(34): validateJWTFromRequest('eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InJhdWxAZXhhbXBsZS5jb20iLCJpYXQiOjE3MDk5MzIzMzIsImV4cCI6MTcxMTczMjMzMn0.k0PQQ0NTBpK9Ux68aoz3cfTDikR_iKKipuyTiOEMP1U')
 4 SYSTEMPATH\Filters\Filters.php(184): App\Filters\JWTAuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 5 SYSTEMPATH\CodeIgniter.php(474): CodeIgniter\Filters\Filters->run('client/', 'before')
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 17:09:40 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@example.com and active = 1' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM apisolve_sandbox.users WHERE email = raul@example.com and active = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM apisolve_sandbox.users WHERE email = raul@example.com and active = 1')
 3 APPPATH\Models\UserModel.php(41): CodeIgniter\Database\BaseConnection->query('SELECT * FROM apisolve_sandbox.users WHERE email = raul@example.com and active = 1')
 4 APPPATH\Helpers\jwt_helper.php(25): App\Models\UserModel->findUserByEmailAddress('raul@example.com')
 5 APPPATH\Filters\JWTAuthenticationFilter.php(34): validateJWTFromRequest('eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InJhdWxAZXhhbXBsZS5jb20iLCJpYXQiOjE3MDk5MzIzMzIsImV4cCI6MTcxMTczMjMzMn0.k0PQQ0NTBpK9Ux68aoz3cfTDikR_iKKipuyTiOEMP1U')
 6 SYSTEMPATH\Filters\Filters.php(184): App\Filters\JWTAuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 7 SYSTEMPATH\CodeIgniter.php(474): CodeIgniter\Filters\Filters->run('client/', 'before')
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 17:09:40 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@example.com and active = 1' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT * FROM apisolve_sandbox.users WHERE email = raul@example.com and active = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM apisolve_sandbox.users WHERE email = raul@example.com and active = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM apisolve_sandbox.users WHERE email = raul@example.com and active = 1')
 4 APPPATH\Models\UserModel.php(41): CodeIgniter\Database\BaseConnection->query('SELECT * FROM apisolve_sandbox.users WHERE email = raul@example.com and active = 1')
 5 APPPATH\Helpers\jwt_helper.php(25): App\Models\UserModel->findUserByEmailAddress('raul@example.com')
 6 APPPATH\Filters\JWTAuthenticationFilter.php(34): validateJWTFromRequest('eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InJhdWxAZXhhbXBsZS5jb20iLCJpYXQiOjE3MDk5MzIzMzIsImV4cCI6MTcxMTczMjMzMn0.k0PQQ0NTBpK9Ux68aoz3cfTDikR_iKKipuyTiOEMP1U')
 7 SYSTEMPATH\Filters\Filters.php(184): App\Filters\JWTAuthenticationFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 8 SYSTEMPATH\CodeIgniter.php(474): CodeIgniter\Filters\Filters->run('client/', 'before')
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 17:09:49 --> Error: Call to undefined method App\Controllers\Users::getJWTForUser()
in APPPATH\Controllers\Users.php on line 25.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Users->auth()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
CRITICAL - 2024-03-11 17:11:29 --> TypeError: getSignedJWTForUser(): Argument #1 ($email) must be of type string, array given, called in C:\xampp\htdocs\apisolve\app\Controllers\Auth.php on line 34
in APPPATH\Helpers\jwt_helper.php on line 28.
 1 APPPATH\Controllers\Auth.php(34): getSignedJWTForUser([...])
 2 APPPATH\Controllers\Auth.php(24): App\Controllers\Auth->getJWTForUser([...], 'Pass!123')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\apisolve\\public\\index.php')
